_posts/en/newsletters/2021-10-20-newsletter.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 'Bitcoin Optech Newsletter #171'
+permalink: /en/newsletters/2021/10/20/
+name: 2021-10-20-newsletter
+slug: 2021-10-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Paying offline LN nodes:** Matt Corallo started a [thread][corallo
+  mobile] on the Lightning-Dev mailing list for brainstorming how
+  frequently-offline LN nodes (such as those on mobile devices) can
+  receive payments without requiring a custodial solution or locking up
+  channel liquidity for prolonged periods of time.  Corallo believes
+  there's a reasonably good solution involving an untrusted third
+  party once LN is [upgraded][zmn taproot] to support [PTLCs][topic
+  ptlc], but he's also seeking suggestions for any alternative solutions
+  that could be deployed even before PTLC support is added.
+
+- **FIXME:harding needs a better title:** a few weeks apart, developers
+  ZmnSCPxj and Joost Jager each made [substantially][zmn prop] similar
+  [proposals][jager prop] for eliminating the requirement to lock up
+  capital in order to probe a payment path.  Both proposals suggest this
+  as a first step towards adding upfront routing fees to the
+  network---fees that would cost the spender money even if a payment
+  attempt failed.  Upfront fees are one of the suggested mitigations for
+  [channel jamming attacks][topic channel jamming attacks].
+
+    Currently, LN nodes can send guaranteed-to-fail payments in order to
+    probe a payment path.  For example, Alice generates an [HTLC][topic
+    htlc] that pays a preimage nobody knows.  She routes the payment
+    through Bob and Charlie to Zed.  Since Zed doesn't know the payment
+    preimage, he's forced to reject the payment even though it indicates
+    he's supposed to be the ultimate receiver.  If Alice receives the
+    reject message from Zed's node, she knows that Bob and Charlie had
+    enough funds in their channels to allow paying Zed, and so she can
+    immediately react to Zed's rejection by sending an actual payment
+    that has a high probability of succeeding (it would only fail if Bob's
+    or Charlie's balances had changed in the meantime).  The advantage
```

Comment:
This isn't the _only_ reason that the payment could fail. For example, Bob or Charlie could go offline, or change their forwarding policy, or ...